extensions/wikia/Chat2/ChatAjax.class.php

```diff
@@ -22,129 +26,143 @@ class ChatAjax {
 	 * If the user is not allowed to chat, an error message is returned (which can be shown to the user).
 	 */
 	static public function getUserInfo() {
```

Comment:
this function has 90 lines, what about splitting it into couple of smaller pieces?